apis/compute/v1alpha1/statefulserverset_types.go

```diff
@@ -133,6 +133,14 @@ type StatefulServerSetReplicaStatus struct {
 	LastModified metav1.Time `json:"lastModified,omitempty"`
 }
 
+// StatefulServerSetVolumeStatus contains the status of a Volume.
+type StatefulServerSetVolumeStatus struct {
```

Comment:
why do we want to add this and not use VolumeStatus?

Reply:
Because VolumeStatus doesn't contain the replica index field, something which is specific to Stateful Server Sets and Server Sets

Reply:
We decided to go with the existing VolumeStatus